### PR TITLE
Set minimum and maximum time range

### LINKED ISSUE #47: Minimum and maximum
Possibility to set minimum and maximum values for the input / dropdown. In case of keyboard navigation start from minimum, go till maximum and loop. In case of dropdown, show only values in between min and max.

### PR DESCRIPTION
**Pull request by pierlon**
Fixes #47.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-time-picker/92)

### CODE REVIEW

**Comment by pierlon** [comment]
Any comments @tomivirkki @yuriy-fix?

**Comment by tomivirkki** [comment]
HI @pierlon, thanks for the contribution. The feature seems to be working quite well. I couldn't find a way to change the min/max values dynamically though. Might be enough to add `min` and `max` to the `__updateDropdownItems(....)` observer arguments to update the dropdown items dynamically. 

Can you also add a demo where the new feature is in use? One demo snippet with min/max values should be enough.

**Comment by pierlon** [comment]
Done :smile:

File: src/vaadin-time-picker.html [hunk 1]
```diff
@@ -567,6 +569,16 @@
           this.__updateValue(parsedObj);
         }
 
+        __minChanged(value) {
+          this.min = value;
```
Comment (web-padawan) [inline src/vaadin-time-picker.html]: this line is not needed, as `value` passed as an argument is already what `this.min` contains

Comment (tomivirkki) [reply src/vaadin-time-picker.html]: Also, are these separate observer methods needed for some reason? What I meant with my previous comment was that you can add `min` and `max` to the [existing multi-property observer](https://github.com/vaadin/vaadin-time-picker/blob/master/src/vaadin-time-picker.html#L326) to have it invoke every time one of these properties change.

Comment (pierlon) [reply src/vaadin-time-picker.html]: Oh didn't know about that :man_facepalming:

**Comment by pierlon** [comment]
Hey @limonte,

I made a slight adjustment to the tests you gave, the `value` property is set before `min` or `max` properties, so I reflected that in the tests.

**Pull request merged**